src/sourabh/Shop.java

```diff
@@ -0,0 +1,100 @@
+package sourabh;
+
+public class Shop {
+	int maggieQty;
+	int dosaQty;
+	int oilPacketsQty;
+	int panipuriQty;
+	int masalaQty;
+	void setInitialStocks(int maggieQtyini, int dosaQtyini, int oilPacketsQtyini, int panipuriQtyini, int masalaQtyini) {
+		maggieQty= maggieQtyini;
+		dosaQty= dosaQtyini;
+		oilPacketsQty= oilPacketsQtyini;
+		panipuriQty= panipuriQtyini;
+		masalaQty= masalaQtyini;
+	}
+
+	void customerPurchase(int maggieQtyCust , int dosaQtyCust, int oilPacketsQtyCust, int panipuriQtyCust, int masalaQtyCust) {
+		System.out.println("*************************");
+		if(maggieQtyCust<= maggieQty) {
+			maggieQty= maggieQty- maggieQtyCust; 
+			System.out.println("Maggie Purchased "+maggieQtyCust);
+		}
+		else{
+			System.out.println("Maggie running out of stock");
```

Comment:
Why we printing the OOS statements here, we have separate requirement for this FR2 and FR3